cmd/kg/main.go

```diff
@@ -125,7 +126,8 @@ var (
 
 func init() {
 	cmd.Flags().StringVar(&backend, "backend", k8s.Backend, fmt.Sprintf("The backend for the mesh. Possible values: %s", availableBackends))
-	cmd.Flags().BoolVar(&cleanUpIface, "clean-up-interface", false, "Should Kilo delete its interface when it shuts down?")
+	cmd.Flags().BoolVar(&cleanUp, "clean-up", false, "Clean up network modifications on shutdown.")
```

Comment:
default should probably be `true` so that the default behaviour is the way it was before?

Reply:
for sure, have been using this setup for quick and dirty testing it on our clusters - will change to `true`

Reply:
done